web_src/less/_base.less

```diff
@@ -384,6 +388,11 @@ a.muted:hover,
   background: var(--color-active);
 }
 
+.ui.secondary.menu.tight .item {
+  padding-left: .85714286em;
+  padding-right: .85714286em;
```

Comment:
what did these mean?

Reply:
Those are simply the same as the padding top/bottom, copied from `semantin.css`